python/desc/imsim/ImageSimulator.py

```diff
@@ -239,9 +240,8 @@ def run(self, processes=1, wait_time=None, node_id=0):
         global CHECKPOINT_SUMMARY
         if (self.file_id is not None and processes > 1 and
             CHECKPOINT_SUMMARY is None):
-            visit = self.obs_md.OpsimMetaData['obshistID']
-            CHECKPOINT_SUMMARY \
-                = CheckpointSummary(db_file='ckpt_{}_{}.sqlite3'.format(visit, node_id))
+            db_file = 'ckpt_{}_{}.sqlite3'.format(self.file_id, node_id)
```

Comment:
What's the rational for using the file_id command line parameter here instead of the visit_id as before (and same question for the the checkpoint)?

Reply:
`visit_id` was too restrictive.  If I want to put additional information in the checkpoint summary file name (e.g., the filter), then `file_id` is the better way to do it.